sdk/nodejs/api/client.gen.ts

```diff
@@ -217,12 +217,12 @@ export type ID = string
 export type Platform = string
 
 export type ClientContainerOpts = {
-  id?: ContainerID | Container
+  id?: Container
```

Comment:
This should be `ContainerID`.

Reply:
Exact. There is some logic that needs to be duplicated to avoid creating the type altogether.

Reply:
Done